tests/testutil/service.go

```diff
@@ -111,6 +138,20 @@ func (ts *TestService) Weight(percentage int) {
 	}
 }
 
+// RolloutWeight calls "rio weight --rollout=true --rollout-increment={increment} --rollout-interval={interval} ns/service:version={percentage}" on this service
+func (ts *TestService) RolloutWeight(increment int, interval int, percentage int) {
```

Comment:
Can we just use func `weight` and pass in params ? It'd be good to keep funcs and cli command as close to 1:1 as possible.

Reply:
Resolved! May want to take a look at the changes I made to fix that. We already had a `func Weight` so I just updated that and its uses to specify rollout or not.